### PR TITLE
Add logging middleware. Configure logging.

### PR DESCRIPTION
**Pull request by pmtischler**
For #36

### CODE REVIEW

File: server/server/settings.py [hunk 1]
```diff
@@ -115,14 +116,25 @@
         },
     },
     'loggers': {
+        'py.warnings': {
```
Comment (prattmic) [inline server/server/settings.py]: What do all of these categories do?

Comment (pmtischler) [inline server/server/settings.py]: These are the named loggers. For example, the named logger we created [here](https://github.com/auvsi-suas/interop/blob/master/server/auvsi_suas/views/__init__.py#L3) is configured by the category below.

Comment (prattmic) [inline server/server/settings.py]: Is there a way to define 'default' settings for loggers not specified here? It seems like we want a file handler for all loggers.

**Comment by prattmic** [comment]
LGTM

File: server/server/settings.py (hunk 1 above)
Comment (pmtischler) [inline server/server/settings.py]: From the [Django docs](https://docs.djangoproject.com/en/1.9/topics/logging/#django):
> django is the catch-all logger. No messages are posted directly to this logger.

My best guess is that my definition of 'django' creates the 'default' logger for all Django components, and 'py.warnings' handles the rest.

**Pull request merged**